llm/llama-4/README.md

```diff
@@ -0,0 +1,144 @@
+
+<!-- $REMOVE -->
+# Run Llama 4 on Kubernetes or Any Cloud
+<!-- $END_REMOVE -->
+<!-- $UNCOMMENT# Llama 4 -->
+
+
+[Llama 4](https://ai.meta.com/blog/llama-4-multimodal-intelligence/) family was released by Meta on Apr 5, 2025.
+
+https://github.com/user-attachments/assets/48cdc44a-31a5-45f0-93be-7a8b6c6a0ded
```

Comment:
This video is great but probably better to have a simple cover photo instead? When presented as the very first thing in the README, the video looks like a wall of text. We can place the video after the first `sky launch` instruction.